### PR TITLE
Default content

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW 

# Summary
- Stanford News Feature

# Review By (Date)
- Thursday April 29th

# Urgency
- High

# Steps to Test

1. You can use: https://github.com/SU-SWS/acsf-lelandd8/pull/13 to test the build and functionality
2. Review the code and leave comments

# Affected Projects or Products
- D8 CORE 
- SOE Profile

### CODE REVIEW

**Comment by boznik** [comment]
@sherakama Is there supposed to be content living at /news with this after a fresh install?

**Comment by sherakama** [comment]
> @sherakama Is there supposed to be content living at /news with this after a fresh install?

Yes, the three nodes and a handful of terms should all be available. I put up a build branch this morning here: https://github.com/SU-SWS/acsf-lelandd8/pull/13

**Comment by sherakama** [comment]
@boznik @jenbreese 

There are likely more code changes to go into the main development branch before all is said and done but this is a good time for code review.

File: stanford_news.module [hunk 1]
```diff
@@ -6,28 +6,28 @@
...
 function stanford_news_theme() {
   return [
-    'node__stanford_news__full' => [
-      'template' => 'node--stanford-news',
-      'base hook' => 'node'
-    ],
     'field__node__su_news_topics' => [
-      'template' => 'field--node--su-news-topics',
+      'template' => 'field/field--node--su-news-topics',
       'base hook' => 'field'
```
Comment (sherakama) [inline stanford_news.module]: @boznik can you confirm that this field template is still needed. There is a todo in the template referencing an issue that was merged in 2018.

**Pull request merged**